deploy/helm/sumologic/values.yaml

```diff
@@ -1867,8 +1895,8 @@ kube-prometheus-stack:
           requests:
             cpu: 1m
             memory: 8Mi
-      containers:
-        - name: "config-reloader"
+      initContainers:
```

Comment:
Are you sure we don't need these env vars for the regular (non-init) `config-reloader` container?

Reply:
fair question, I will check it

Reply:
You're right. I'm curious if I should write migration which copies config-relader config as init-config-reloader

Reply:
We agreed to do nothing